testgrid/config/config.yaml

```diff
@@ -3567,6 +3599,8 @@ dashboards:
     test_group_name: ci-kubernetes-node-kubelet-serial-cpu-manager
     alert_options:
       alert_mail_to_addresses: 'balaji.warft@gmail.com, connor.p.d@gmail.com'
+  - name: ci-cri-containerd-e2e-gce-device-plugin-gpu
+    gcs_prefix: kubernetes-jenkins/logs/ci-cri-containerd-e2e-gce-device-plugin-gpu
```

Comment:
Sorry, I copied the wrong line in my earlier comment.
This should be:
```
test_group_name: ci-cri-containerd-e2e-gce-device-plugin-gpu
```

Reply:
oops........wrong copy